### PR TITLE
docs: create 'Add new boards' and 'Build tags' pages

### PR DESCRIPTION
**Pull request by deadprogram**
Create 'Add new boards' and 'Build tags' pages in docs to allow 'Add new boards' wiki page to be removed.

### CODE REVIEW

File: docs/adding-new-boards.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
+.. _installation:
```
Comment (aykevl) [inline docs/adding-new-boards.rst]: This is the identifier for the page to link to it, so it must be unique.

File: docs/buildtags.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.. _installation:
```
Comment (aykevl) [inline docs/buildtags.rst]: same here

Comment (deadprogram) [reply docs/buildtags.rst]: Also fixed.

**Comment by deadprogram** [comment]
I think both of those issues have now been corrected.

File: docs/adding-new-boards.rst [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+First make sure the particular chip and chip family is supported, see
+below.
+
+Adding a new board is relatively easy. You will need:
+
+-  A new target description in .json format. It is probably easiest to
+   copy an existing description from the same chip and adjust it to your
+   needs.
+   TODO: make it possible to inherit these definitions from a base .json
+   file to avoid duplication.
```
Comment (aykevl) [inline docs/adding-new-boards.rst]: Can you add a link somewhere here to the build tags documentation? With the build tags docs on a separate page it's not that obvious anymore that they're related. A link can be added with for example ``:ref:`build tags <buildtags>` ``. You can say something like "For more information on the `build-tags` field, see <build tags>" (or however you'd like to formulate that).

Also, maybe this TODO needs to be removed. It was mostly a hint to myself. Such a thing doesn't really belong in the documentation.

Comment (deadprogram) [reply docs/adding-new-boards.rst]: The PR seems like it is turning into more work then you having updated the pages yourself, which was certainly not my intention. Maybe we should just forget the whole idea, and just do nothing at present?

**Comment by deadprogram** [comment]
Going to close, since appears is becoming hassle and not really helpful contribution.

**Pull request closed without merging**